<launch>
   <node name="roboclaw_driver" pkg="roboclaw_driver" type="roboclaw_driver" output="screen">
		<param name="port" value="/dev/ttyACM0" />
		<param name="baud" value="1000000" />
		<param name="rate" value="30" />
		<param name="base_width" value="0.265" />
		<param name="ticks_per_metre" value="21738" />
		<param name="base_frame_id" value="base_footprint" />	
		<param name="QPPS" value="11600" />
   </node>
</launch>
